lib/backends/sdc/containers.js

```diff
@@ -2017,15 +2476,35 @@ function getContainers(opts, callback) {
             });
 
             if (filterErr) {
-                callback(filterErr);
+                cb(filterErr);
                 return;
             }
+
+            stash.objects = objects;
+            cb();
+        }, function _limitResults(stash, cb) {
+            var objects = stash.objects;
+
+            if (opts.limit > 0) {
+                objects = objects.slice(0, opts.limit);
+            }
+
+            // last cb in chain, so we pass objects to results
+            cb(null, objects);
         }
+    ]}, function _pipelineCb(p_err, p_results) {
```

Comment:
Yes, the p_ prefixes are to avoid overshadowing the variables in the outer scope. I'll take a look here and see what I can do.